M2/Macaulay2/m2/expressions.m2

```diff
@@ -550,25 +550,46 @@ toString'(Function, SparseMonomialVectorExpression) := (fmt,v) -> toString (
 -----------------------------------------------------------------------------
 MatrixExpression = new HeaderType of Expression
 MatrixExpression.synonym = "matrix expression"
-matrixOpts1 := new OptionTable from {Blocks=>null,Degrees=>null,MutableMatrix=>false};
+matrixOpts1 := new OptionTable from {
+    Blocks => null,
+    Degrees => null,
+    MutableMatrix => false,
+    zero => null};
```

Comment:
any reason `zero` is the only lowercase option? and incidentally which type is this `zero`?

Reply:
I went with `zero` since it's already exported.  When it's non-null, it will be a sequence of two modules (the target and source).

Reply:
I meant, what is the class of `zero` itself, not the option labelled zero.
usually it's nice if all options have the same type; which one is a matter of taste, it seems (all symbols seems the most common choice, though there's serious overhead in doing that; I would think all strings would actually make more sense, but it's rarely used. I digress)

Reply:
This is an excellent question!  I thought I'd put `symbol zero` here, but I didn't.  And `zero` is a function closure, which *should* have raised an error...

```m2
i1 : new OptionTable from {zero => null}
stdio:1:0:(3): error: expected option key to be a symbol, a string, or a type
```

Reply:
Ohhh I figured it out -- `zero` is still a symbol at this point since it's not defined as a function until integers.m2, which is later in the load sequence.  (Same with `MutableMatrix` -- it's a type defined in mutablemat.m2, but it's a symbol here).

I'll change this so they're both explicitly symbols, just in case any code gets moved around